src/WTG.BulkAnalysis.Core/Processor.cs

```diff
@@ -119,7 +119,17 @@ static async Task<Workspace> LoadSolutionIntoNewWorkspace(RunContext context, st
 
 		static void ConfigureWorkspace(Workspace workspace)
 		{
-			workspace.Options = workspace.Options
+			Solution solution;
+
+			do
+			{
+				solution = workspace.CurrentSolution;
+			}
+			while (!workspace.TryApplyChanges(ConfigureSolution(solution)));

```

Comment:
Why the change here to loop until it sticks?

Reply:
The docs say it can return false if the new solution is not based on the latest solution. It shouldn't really happen since we set this up fairly soon after loading the solution and before we pass it no other threads ... but the alternative would be to ignore the result.

Reply:
If it shouldn't happen then I'd be tempted to just whack a `Debug.Assert` on the boolean.

Reply:
maybe log it as an error?